discovery-provider/plugins/pedalboard/apps/relay/src/server.ts

```diff
@@ -0,0 +1,41 @@
+import App from "basekit/src/app";
+import Fastify from "fastify";
+import { TypeBoxTypeProvider } from "@fastify/type-provider-typebox";
+import { healthCheck } from "./routes/health";
+import { relayHandler } from "./routes/relay";
+import {
+  RelayRequest,
+  RelayRequestType,
+  RelayResponse,
+  RelayResponseType,
+} from "./types/relay";
+import { SharedData } from ".";
+
+export const webServer = async (app: App<SharedData>) => {
+  const fastify = Fastify({
+    logger: true,
+  }).withTypeProvider<TypeBoxTypeProvider>();
```

Comment:
Fastify seemed interesting and is built around async so thought i'd give it a try, might move to something that's more zod friendly though.

Reply:
i wouldn't think so as long as they're mostly async fns, nothing in identity seemed to difficult to port

Reply:
yep! there's two tickets, one for each of those